src/Helpers/email_helper.php

```diff
@@ -41,7 +41,7 @@ function emailer(array $overrides = []): Email
         }
 
         /** @var Email $email */
-        $email = service('email');
+        $email = service('email', false);
```

Comment:
Let's follow MGater's opinion and undo this here.

```suggestion
        $email = service('email');
```

Reply:
So, is the email still going to be `clear()`ed ?

Reply:
See my comment below.

Reply:
Ok. 
Done that now.